book/src/paper-wallet/README.md

```diff
@@ -0,0 +1,21 @@
+# Paper Wallet
+
+This document describes how to create and use a paper wallet with the Solana CLI tools.
+
+{% hint style="info" %}
+We do not intend to advise on how to *securely* create or manage paper wallets. Please research the security concerns carefully.
+{% endhint %}
+
+## Overview
+
+Solana provides a key generation tool to derive keys from BIP39 compliant seed phrases. Solana CLI commands for running a validator and staking tokens all support keypair input via seed phrases.
```

Comment:
80 chars per line, or whatever limit we need to prevent wrapping in the github diff please 🚓

Reply:
Oh! I guess "always wrap" mean always break to 80 chars per line?

Reply:
Yeah, it's difficult to view/comment on long lines in github diffs so that's my main intent with wrapping